variables/ageCalculator.js

```diff
@@ -1 +1,5 @@
-// Enter code below
\ No newline at end of file
+var b_year = 1987;
+var f_year = 2017;
```

Comment:
These variable names could be more descriptive e.g. `birthYear` & `futureYear` is much more clear as to what they are meant to represent